pom.xml

```diff
@@ -44,6 +44,8 @@
         <node.version>v7.1.0</node.version>
         <yarn.version>v0.21.2</yarn.version>
         <junit.version>4.12</junit.version>
+        <spark.version>2.5.5</spark.version>
+        <servlet-api.version>3.1.0</servlet-api.version>
```

Comment:
We can remove this variable if we don't need javax.servlet-api in the new module